freqtrade/strategy/interface.py

```diff
@@ -173,6 +190,60 @@ def shutdown(self, *args, **kwargs):
 
             self.freqai = DummyClass()  # type: ignore
 
+    def setup_freqai_spice_rack(self, config: dict) -> Dict[str, Any]:
+        import difflib
+        import json
+        from pathlib import Path
+        auto_config = config.get('freqai_config', 'lightgbm_config.json')
+        with open(Path('freqtrade') / 'freqai' / 'spice_rack'
+                  / auto_config) as json_file:
+            freqai_config = json.load(json_file)
```

Comment:
This is wrong / will not work depending on the setup. 
it's assuming you run freqtrade from the repository's root directory.

This is probably true for many users - but it's not a must - you can install freqtrade in any environment (also via `pip install freqtrade`) - so the directory "freqtrade" may not be available in the CWD.

You'll have to navigate to that via `Path(__file__).parent` - although i'm not 100% sure that'll work from this location - as you're in the strategy interface - which means the location of "__file__" may be the actual place of the strategy.

I think the best thing would be to move this method to freqai/utils (or a separate file in spice_rack) - and pass in what's needed (most likely just the dataprovider or the exchange object).
That'll also make the "relative path navigation" easier.

Reply:
Good catch, I will try to work it out.

Reply:
This should be resolved in latest commits.